tests/components/zwave_js/test_switch.py

```diff
@@ -286,7 +286,11 @@ async def test_config_parameter_switch(
     client.async_send_command.side_effect = FailedZWaveCommand("test", 1, "test")
 
     # Test turning off error raises proper exception
-    with pytest.raises(HomeAssistantError):
+    with pytest.raises(HomeAssistantError) as err:
         await hass.services.async_call(
             DOMAIN, SERVICE_TURN_OFF, {"entity_id": switch_entity_id}, blocking=True
```

Comment:
```suggestion
            DOMAIN, SERVICE_TURN_OFF, {ATTR_ENTITY_ID: switch_entity_id}, blocking=True
```

Reply:
We don't use that in this module yet, and I'd rather not change it here now.